docs/Data/Semiring/Boolean.md

```diff
@@ -0,0 +1,28 @@
+## Module Data.Semiring.Boolean
+
+#### `BooleanSemiring`
+
+``` purescript
+newtype BooleanSemiring a
+  = BooleanSemiring a
+```
+
+Provides the semiring for any boolean algebra.
```

Comment:
@paf31 You're right; I had been thinking about doing it, but for some reason it didn't seem to fit. But now that I think of it, there really isn't any reason to not do it, so I'll close this PR and do it in `monoids` as I originally proposed.